plugins/plugin-template/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  DsgContext,
+  AmplicationPlugin,
+  Events,
+  Module,
+  EventNames,
+  CreateServerParams,
+  CreateAdminUIParams,
+} from "@amplication/code-gen-types";
+
+
+class ExamplePlugin implements AmplicationPlugin {
+  /**
+   * this is mandatory function that return object with events name. each event can have before or/and after
```

Comment:
I think we can add a link to the list of all events on the docs site - consult with @MoFoGo